src/refiners/foundationals/latent_diffusion/model.py

```diff
@@ -20,7 +20,7 @@ def __init__(
         clip_text_encoder: fl.Chain,
         solver: Solver,
         device: Device | str = "cpu",
-        dtype: DType = torch.float32,
+        dtype: DType | None = torch.float32,
```

Comment:
We shouldn't change `foundationals` API in a PR about training_utils (they must be flexible enough to accommodate).

Reply:
Hmm, is there a way to initialize a StableDiffusion_1 or XL without setting a Dtype? It can prob be a subject of a different pr if needed